F50/Preferences/Preferences.android.cs

```diff
@@ -16,82 +16,64 @@ public bool ContainsKey(string key)
 			{
 				using (var sharedPreferences = GetSharedPreferences())
 				{
-					if (sharedPreferences == null)
-						return false;
-
 					return sharedPreferences.Contains(key);

```

Comment:
What if it is null? Should we do sharedPreferences?.Contains(key) ?? false;

Reply:
There's nothing in the docs that suggests this should actually ever be null... It looks like worst case if you pass a null context into GetSharedPreferences(..) you'll get a java side null pointer exception thrown...

Reply:
Well, we return null if the context is null....

Reply:
We should probably throw an exception in that case with some reasonable explanation to make sure the app has been initialized.. I'm not sure Application.Context should ever be null?